frontend/src/pages/modelServing/screens/projects/InferenceServiceModal/ManageInferenceServiceModal.tsx

```diff
@@ -100,6 +101,21 @@ const ManageInferenceServiceModal: React.FC<ManageInferenceServiceModalProps> =
     return isConnectionValid;
   };
 
+  React.useEffect(() => {
+    if (createData.name && createData.name !== inferenceServiceNameDesc.name) {
+      setInferenceServiceNameDesc('name', createData.name);
```

Comment:
I don't see why we need to add this additional `setInferenceServiceNameDesc` - I thought the issue here was just the spinner rendering and the form state was already getting set eventually?

Reply:
Interesting. The whole way we sync state with these components is a bit wonky, but this approach makes sense with what we have. Thanks